contracts/Utils/ReentrancyGuard.sol

```diff
@@ -14,7 +14,7 @@ contract ReentrancyGuard {
         _notEntered = true; // get a gas-refund post-Istanbul
     }
 
-    constructor() internal {
+    constructor() {
```

Comment:
Looks like we need to lint

Reply:
ts file should be fixed by dbb14aa, sol files seem fine 🤔